### ISSUE TITLE
Incl. Overclock tool

### ISSUE THREAD

**Issue by TuKo1982** [shadow2560/switch_AIO_LS_pack#3]
Would it be possible to incluse an overclock tool in next release ?

Since ppl widely overclock RAM now for TOTK, it would make sense.

**Comment by TuKo1982** [comment]
Have you had time to think about it ? :-)

**Comment by shadow2560** [comment]
Yes and I think I will not include it, the recent update caused problem with the old version of the module. The module has been fixed relatively wuickly but it's a problem to include something so sencible.

An other problem is that I don't want to be accused to include a module that could damage the console, those who want to do such things must do it by themselves, like that they must take the theirs responsabilities.

So for now I will not include the module but maybe one day I will propose to custom the pack into the homebrew AIO_LS_pack_Updater, it's something that I have in mind since some time, I just don't know how to integrate this for now because it could be difficult to maintain and it's not so easy to integrate this type of functions.
